testentry.sh

```
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+set +u
+if [ ! -z "$TEST" ] ; then
+	set -u
+	RETRIES=0
+	MAX_RETRIES=48 # 240s
+	while true ; do
+		if [ "$RETRIES" -gt "$MAX_RETRIES" ] ; then
+			sh -c "sleep2; tmux send-keys \"echo 'exceeded maximum retries waiting for server to become available, tests are likely to fail!'\" Enter" &
```

typo `sleep2`?

---

Good catch. I think I didn't notice this since `sh` by default ignores errors silently, and I didn't use `set -e` in the subshell. 🤷